src/ngraph/frontend/onnx_import/op/pad.cpp

```diff
@@ -73,6 +106,56 @@ namespace ngraph
                 }
 
             } // namespace set_1
+            namespace set_11
+            {
+                NodeVector pad(const Node& node)
+                {
+                    auto data = node.get_ng_inputs().at(0);
+                    auto pads = node.get_ng_inputs().at(1);
+                    std::shared_ptr<ngraph::Node> values;
+
+                    if (node.get_ng_inputs().size() == 3)
+                    {
+                        values = node.get_ng_inputs().at(2);
+                    }
+                    else
+                    {
+                        values = ngraph::op::Constant::create(
+                            data->get_element_type(), ngraph::Shape{}, {0});
+                    }
+
+                    auto axis = ngraph::op::Constant::create(element::i64, ngraph::Shape{}, {0});
+                    NodeVector padding = builder::split(pads, 2, 0);
+                    debugvS(padding.at(0));
+                    debugvS(padding.at(1));
+                    auto padding_begin =
+                        std::make_shared<default_opset::Convert>(padding.at(0), element::i64);
+                    auto padding_end =
+                        std::make_shared<default_opset::Convert>(padding.at(1), element::i64);
+
+                    std::string mode = node.get_attribute_value<std::string>("mode", "constant");
+                    ngraph::op::PadMode pad_mode;
+                    if (mode == "constant")
```

Comment:
Please don't repeat the code from above. Move this to a helper function.

Reply:
Done

Reply:
Done